VERSIONING.md

```diff
@@ -0,0 +1,9 @@
+# Versioning
+
+The NMPolicy versioning follows the [semVer specification](https://semver.org/#semantic-versioning-specification-semver),  
+with major version to breaking changes, 
+minor versions to main branch releases and patch/bugfix versions for main bugfix or release branches versions.
+
+After releasing pushing the new tag to the repo a release branch with the format "release-[major].[minor]" 
+can be created in case something need to be fixed for a specific version, 
```

Comment:
I would change this to "must be created", as we suffered issues of misaligned tags due to tagging patch releases on main in the past.